demisto_sdk/commands/lint/resources/pylint_plugins/base_checker.py

```diff
@@ -0,0 +1,72 @@
+from pylint.checkers import BaseChecker
+from pylint.interfaces import IAstroidChecker
+
+# You can find documentation about adding new checker here:
+# http://pylint.pycqa.org/en/latest/how_tos/custom_checkers.html#write-a-checker
+
+
+class CustomBaseChecker(BaseChecker):
+    __implements__ = IAstroidChecker
+    name = "base-checker"
+    priority = -1
+    msgs = {
+        "W9001": (
+            # displayed-message shown to user
+            "Sys.exit use is found, Please use return instead.",
+            # message-symbol used as alias for message-id
+            "sys-exit-exists",
+            # message-help shown to user when calling pylint --help-msg
+            "Ensure to not use sys.exit in the code.",
+        ),
+        "W9002": (
+            # displayed-message shown to user
+            "Print is found, Please remove all prints from the code.",
+            # message-symbol used as alias for message-id
+            "print-exists",
+            # message-help shown to user when calling pylint --help-msg
+            "Please remove all prints from the code.",
+        ),
+        "W9003": (
+            # displayed-message shown to user
+            "demisto.log is found, Please remove, you can choose to replace it with demisto.debug or logger",
+            # message-symbol used as alias for message-id
+            "demisto-log-exists",
+            # message-help shown to user when calling pylint --help-msg
+            "Please remove all demisto.log from the code.",
+        ),
+        "W9004": (
+            # displayed-message shown to user
+            "time.sleep is found, Please remove any sleep functionality from the code",
+            # message-symbol used as alias for message-id
+            "sleep-exists",
+            # message-help shown to user when calling pylint --help-msg
+            "Please remove all sleep functionality from the code.",
```

Comment:
this can be commented out for now, right?

Reply:
yes, removed it for now